pettingzoo/utils/wrappers/terminate_illegal.py

```diff
@@ -9,23 +12,23 @@ class TerminateIllegalWrapper(BaseWrapper):
         illegal_reward: number that is the value of the player making an illegal move.
     """
 
-    def __init__(self, env, illegal_reward):
+    def __init__(self, env: AECEnv, illegal_reward: float) -> None:
         super().__init__(env)
         self._illegal_value = illegal_reward
         self._prev_obs = None
 
-    def reset(self, seed=None, options=None):
+    def reset(self, seed: int | None = None, options: dict | None = None) -> None:
         self._terminated = False
         self._prev_obs = None
         super().reset(seed=seed, options=options)
 
-    def observe(self, agent):
+    def observe(self, agent: str) -> ObsType | None:
```

Comment:
Can it be none?

Reply:
Yeah it can be None, just one of the more diabolical things.